File: Programing-Basics-With-Python/0-5-For-Loops/Exercise/salary.py
open_browser_tabs_count = int(input())
salary = int(input())
is_salary_lost = False

for i in range(open_browser_tabs_count):
    website = input()

    if website == "Facebook":
        salary -= 150
    elif website == "Instagram":
        salary -= 100
    elif website == "Reddit":
        salary -= 50

    if salary <= 0:
        is_salary_lost = True
        print(f"You have lost your salary.")
        break

if not is_salary_lost:
    print(f"{salary}")

File: JS-Basics/0-4-For-Loops/Exercise/salary.js
function salary(input) {
    let tabsCount = Number(input.shift());
    let salary = Number(input.shift());
    for (let i = 0; i < input.length; i++) {
        let site = input[i];
        if (site == "Facebook") {
            salary -= 150;
        } else if (site == "Instagram") {
            salary -= 100;
        } else if (site == "Reddit") {
            salary -= 50
        }

        if (salary <= 0) {
            console.log("You have lost your salary.");
            return;
        }
    }
    console.log(`${salary.toFixed(0)}`);
}


salary(["10", "750", "Facebook", "Dev.bg", "Instagram", "Facebook", "Reddit", "Facebook", "Facebook"])
salary(["3", "500", "Github.com", "Stackoverflow.com", "softuni.bg"])
